_includes/configuration/style-formats.md

```diff
@@ -3,9 +3,10 @@
 
 This option allows you to define custom items for the `styleselect` dropdown toolbar button and the `formats` menu item.
 
-There are three types of items the array can contain:
+There are four types of items the array can contain:
 
-* **Style:** The item must have a `title` and either a `format` with refers to a pre-registered editor [format]({{ site.baseurl }}/configure/content-formatting/#formats) or the necessary fields to [specify a new format]({{ site.baseurl }}/configure/content-formatting/#formattype). The item will be rendered in the UI as a clickable item that applies the given format.
+* **Style:** The item must have a `title` and an optional `name` to specify what it will be registered as. Then it needs the other necessary fields to [specify a new format]({{ site.baseurl }}/configure/content-formatting/#formattype). If the name is specified it will be prefixed with `custom-` this is to ensure that it doesn't clash with the default editor formats, if you give it a unique name you can then refer to it using the formatting api. The item will be rendered in the UI as a clickable item that applies the given format.
```

Comment:
```suggestion
* **Style:** The item must have a `title` and either a `format` which refers to a pre-registered editor [format]({{ site.baseurl }}/configure/content-formatting/#formats) or has the necessary fields to [specify a new format]({{ site.baseurl }}/configure/content-formatting/#formattype). An optional `name` can be provided for the style, which will be prefixed with `custom-` to ensure that it does not override the default editor formats. if given a unique `name`, the style can be used with the formatting API. The item will be rendered in the UI as a clickable item that applies the given format.
```

Reply:
Does this mean I should get rid of the the extra style reference item I added. I thought it was better to have that as a separate thing since it otherwise talks about two concept one being a style with format parameters and one being a reference to an existing format.

Reply:
Sorry, no, my fault. I've updated the suggestion to hopefully address that misunderstanding. 

Also, I pushed a commit to update how the live demos render and to add a simple example. The live demos are from the example/demo pages and may be a little too "noisy".

Reply:
Sorry, no, my fault. I've updated the suggestion to hopefully address that misunderstanding. 

Also, I pushed a commit to update how the live demos render and to add a simple example. The live demos are from the example/demo pages and may be a little too "noisy".

Reply:
Ok. Cool thanks. I updated the text now.